cli/dcoscli/main.py

```diff
@@ -69,19 +69,22 @@ def _main():
 
     util.configure_process_from_environ()
 
-    if config.uses_deprecated_config():
-        cluster.move_to_cluster_config()
-
     if args['--version']:
         return _get_versions()
 
     command = args['<command>']
 
     if not command:
-        if args['--help']:
-            command = "help"
-        else:
-            return dcos_help()
+        return dcos_help()
+
+    if constants.DCOS_CONFIG_ENV in os.environ:
+        msg = ('{} is deprecated, please consider using '
+               '`dcos cluster setup <dcos_url>`.')
+        err = errors.DefaultError(msg.format(constants.DCOS_CONFIG_ENV))
```

Comment:
Does this mean that this warning is going to be printed for every command that gets issued so long as this environment variable is set? That seems like a little much.

Reply:
Yes, should I make this printed only before the migration occurs?

Reply:
That seems more reasonable to me. I'd also print a longer message stating how this variable will now be ignored going forward, and that DCOS_DIR is the replacement for relocating where the (now) managed configs live. It also seems like we should probably prompt to verify that the user has read this warning and is OK with migrating.

Reply:
I think I still miss this which isn't implemented when `DCOS_CONFIG` is set : 
> After `dcos cluster setup` is called (or automatic conversion has occurred),
   update the ‘active’ cluster’s config whenever old flow commands are called
   (print different warning message saying the calls are deprecated and cluster config state may now be corrupted)